_docs/tasks/rate-limiting.md

```diff
@@ -44,28 +47,79 @@ Using Istio we can ensure that `1qps` is not breached.
    If you log in as any other user (or logout) you should see red ratings stars with each review,
    indicating that the `ratings` service is being called by the "v3" version of the `reviews` service.
 
-1. Configure mixer with the rate limit. 
-
-   Save this as ratelimit.yaml: 
+1. Configure memquota adapter with rate limits. 
```

Comment:
should we use "``"s around memquota?  `memquota` ?

Should this read: `Configure a 'memquota' handler with rate limits` ?